src/vcpkg/remove.cpp

```diff
@@ -52,7 +51,7 @@ namespace vcpkg::Remove
                 const auto status = fs.symlink_status(target, ec);
                 if (ec)
                 {
-                    print2(Color::error, "failed: symlink_status(", target, "): ", ec.message(), "\n");
+                    msg::println_error(msgFailedSymlinkStatus, msg::path = target, msg::error_msg = ec.message());
```

Comment:
This should use the same formatting mechanism as exit_filesystem_call_error (and FailedSymlinkStatus deleted)

Reply:
Ditto other filesystem operations here

Reply:
I'm not sure what you mean. Can you elaborate?

Reply:
In `files.cpp` there's a function `exit_system_call_error` which forms an error message out of the name of the called operation, the parameters, and the actual error returned by the operating system, which notably means no localization is necessary. (It is done by the operating system) This comment is asking to extract that and print that instead rather than making slightly different localized message for each "generic filesystem operation failed" message.

https://github.com/microsoft/vcpkg-tool/blob/da270f4f9b733d2bc37f33a5a5d0d439bce55524/src/vcpkg/base/files.cpp#L76-L77